spec/models/restaurant_spec.rb

```diff
@@ -25,4 +25,27 @@
       expect(FactoryGirl.create(:restaurant)).to be_valid
     end
   end
+
+  describe 'Geocoder' do
+    before(:all) do
+      Geocoder.configure(:lookup => :test)
+      Geocoder::Lookup::Test.add_stub(
+        "Ostrahamngatan 5,", [
+          {
+            'latitude' =>   57.7093164,
+            'longitude' =>  11.9663941,
+            'address'   => 'Ostrahamngatan 5,',
+            'state'     => 'Vastra Gotaland',
+            'city'      => 'Gothenbourg',
+            'country'   => 'Sweden',
+          }
+        ]
+      )
+    end
```

Comment:
would appreciate help, when I moved this block to the rails_helper file, I am unable to make the test pass except this block remains in the spec test file.